templates/terraform/examples/app_engine_flexible_app_version.tf.erb

```diff
@@ -1,3 +1,9 @@
+resource "google_project_service" "service" {
+  service = "appengineflex.googleapis.com"
+
+  disable_dependent_services = true
```

Comment:
Can you set `disable_on_destroy` to `false` and remove `disable_dependent_services`? That way, TF will enable the service if possible and won't turn it back off (possibly conflicting if tests run at the same time). Plus, I think that's what most users will expect.